packages/react-components/react-field/src/components/Field/useFieldStyles.ts

```diff
@@ -59,12 +59,18 @@ const useSecondaryTextStyles = makeStyles({
   },
 });
 
-const useValidationMessageIconStyles = makeStyles({
+const useValidationMessageStyles = makeStyles({
   base: {
+    display: 'flex',
+  },
```

Comment:
The `useSecondaryTextStyles` is the one that's used for the validation message (and hint) styles. You should put this there instead, and keep this one for the icon only (revert the name to `useValidationMessageIconStyles`).

Reply:
Just commenting here what we discussed in chat:

We're switching to grid from flexbox, since that allows the icon to be in its own column without needing to wrap it in another node. The reason for moving away from `display: flex` is that it interferes with layout of nested items, e.g. links within hint or validation message text.